utils/extract_subset.py

```diff
@@ -0,0 +1,48 @@
+import sys
+import os
+import argparse
+import json
+
+
+def get_args():
+  parser = argparse.ArgumentParser(description="""
+      This script is used to process raw json dataset of GigaSpeech
+      to download selected subset.
+      """)
+  parser.add_argument('subset', type=str, choices=['DEV', 'TEST', 'XL', 'L', 'M', 'S', 'XS'], default='XL', help="""Subset to download""")
+  parser.add_argument('input_json', help="""Input json file of Gigaspeech""")
+  args = parser.parse_args()
+  return args
+
+
+def meta_analysis(input_json, subset):
+    subset_list=['{XS}', '{S}', '{M}', '{L}', '{XL}', '{TEST}', '{DEV}']
```

Comment:
We added `choices` for the parameter `subset` so this is probably not necessary? Please test.